.github/workflows/testdriver.yml

```diff
@@ -0,0 +1,28 @@
+name: TestDriver.ai
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: "TestDriver"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: testdriverai/action@main
+        with:
+          version: v4.0.44
+          key: ${{secrets.TESTDRIVER_API_KEY}}
+          prerun: |
+            npm install
+            npm run build
+            npm install dashcam-chrome --save
+            Start-Process "C:/Program Files/Google/Chrome/Application/chrome.exe" -ArgumentList "--start-maximized", "--load-extension=$(pwd)/node_modules/dashcam-chrome/build", "https://stirlingpdf.io"
+            Start-Sleep -Seconds 20
+          prompt: |
+            1. /run testdriver/test.yml
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why does it need GitHub token here of it doesn't connect with github

Reply:
> Why does it need GitHub token here of it doesn't connect with github

In order to run PR code, it clones the repo down onto the VM.

We then set up the repo using the `prerun` script
https://docs.testdriver.ai/continuous-integration/prerun-scripts

Reply:
Oh sorry, actually since the repo is OSS the GH_TOKEN is optional. It will use our 'testdriverbot' token instead.